python/foglamp/services/core/server.py

```diff
@@ -617,10 +617,13 @@ def start(cls):
 
     @classmethod
     def stop(cls, loop=None):
-        if loop is None:
-            loop = asyncio.get_event_loop()
-        loop.run_until_complete(cls._stop())
-        loop.stop()
+        try:
+            if loop is None:
+                loop = asyncio.get_event_loop()
+            loop.run_until_complete(cls._stop())
+            loop.stop()
+        except RuntimeError as ex:
+            _logger.exception("Error while stopping foglamp server: {}".format(str(ex)))
```

Comment:
foglamp -> FogLAMP
RuntimeError as `ex` > RuntimeError as `e`

Reply:
I think we should also raise, not only just log; this is why _stop has bare exception raised!